addons/web/static/src/js/views/kanban/kanban_model.js

```diff
@@ -292,6 +292,13 @@ var KanbanModel = BasicModel.extend({
         if (element.parentID) {
             element.limit = this.loadParams.limit;
         }
+        // if list datapoint is reloaded then remove activeFilter from all child datapoint
```

Comment:
Sorry but I do not think that it is the good fix.
See https://youtu.be/ZN8eeX0ioTk